web/storybook.tiltfile

```diff
@@ -14,6 +14,33 @@ local_resource(
   readiness_probe=probe(http_get=http_get_action(port=9009)),
   labels=["frontend"])
 
+cmd_button(
+  name='2prettier',
+  resource='storybook',
+  argv=['sh', '-c', 'cd .. && make prettier'],
+  text='2. Run prettier',
+  location=location.RESOURCE,
+  icon_name='cleaning_services',
+)
+
+cmd_button(
+  name='1ts-compile',
+  resource='storybook',
+  argv=['sh', '-c', 'node_modules/.bin/tsc -p .'],
+  text='1. Verify compile',
+  location=location.RESOURCE,
+  icon_name='handyman',
+)
+
+cmd_button(
+  name='3check-js',
+  resource='storybook',
+  argv=['sh', '-c', 'cd .. && make check-js'],
```

Comment:
The underlying `cmd` already supports a dir, but the uibutton extension just hardcodes it to `config.main_dir`. We should probably allow the user to set that so they don't have to do weird stuff like that - any interest in taking a stab at that? shall I?

Reply:
oh yeah! i can take a stab at that.